README.md

```diff
@@ -29,23 +31,31 @@ Make sure to allow incoming connections with TCP on the port of the control serv
 `31066`) and allow all incoming UDP connections on the ports of the border routers (for AS `1-ff00:0:133` this is
 `31068` and `31070`).
 
-## Building
+## Building and Installation
 
-Setup the CMake build with
+Setup the CMake build directory in `./dist` with
 
 ```bash
 cmake -DBUILD_EXAMPLES=ON -DBUILD_TESTS=ON -DBUILD_DOCS=ON -B dist
 ```
 
-The options `DBUILD_EXAMPLES`, `DBUILD_TESTS` and `DBUILD_DOCS` can also be turned off.
+The options `DBUILD_EXAMPLES`, `DBUILD_TESTS` and `DBUILD_DOCS` can be turned off.
```

Comment:
I used `cmake -DBUILD_EXAMPLES=OFF -DBUILD_TESTS=ON -DBUILD_DOCS=ON -B dist` (examples disabled), however, I still get 
```
CMake Error: File [...]/csnet/examples/topology.json does not exist.
```

Reply:
This is because the tests need to build some of the examples.

Reply:
> This is because the tests need to build some of the examples.
Why do the tests need examples?

Also, under **Requirements** it is suggested that the topology file is only required for `Building and running the examples`.

Can we sensibly separate these things and have a pure library build (without examples) without requiring a topology file? Or would that not make sense?

Reply:
Well, if you configure the build with `cmake -DBUILD_EXAMPLES=OFF -DBUILD_TESTS=OFF -DBUILD_DOCS=OFF -B dist` you are only building the library.

Reply:
I assume the `BUILD_EXAMPLES=OFF` disables only some off the examples because others are required for the tests? 
Or does it disable all of them and BUILD_TESTS will fail?

Maybe we can clarify in the doc that `BUILD_TESTS` depends on some of the examples and whether these examples will be build regardless of the BUILD_EXAMPLES option?

Reply:
Yes, setting `BUILD_EXAMPLES=OFF` and `BUILD_TESTS=OFF` will lead to some of the example still being built. I can clarify this in the docs.

Reply:
Great, thanks!